addons/mail/static/src/discuss/core/common/suggestion_service_patch.js

```diff
@@ -8,8 +8,8 @@ const commandRegistry = registry.category("discuss.channel_commands");
 
 /** @type {SuggestionService} */
 const suggestionServicePatch = {
-    getSupportedDelimiters(thread) {
-        const res = super.getSupportedDelimiters(thread);
+    getSupportedDelimiters(thread, env) {
+        const res = super.getSupportedDelimiters(thread, env);
```

Comment:
Nitpick: I would `...arguments` to not have to change whenever the signature changes, more likely for patches to not break original code.